_includes/api/en/4x/express.static.md

```diff
@@ -38,8 +38,6 @@ Possible values for this option are:
 - "deny" - Deny a request for a dotfile, respond with `403`, then call `next()`.
 - "ignore" - Act as if the dotfile does not exist, respond with `404`, then call `next()`.
 
-**NOTE**: With the default value, it will not ignore files in a directory that begins with a dot.
```

Comment:
It's definitely confusing, but this was changed to ignore in a 2.0 beta: https://github.com/expressjs/serve-static/blob/87c5f092ed7cd23202f699b08c7f407c2b536df0/HISTORY.md?plain=1#L27

However, I believe Express 4.x still uses the 1.x release of `serve-static`: https://github.com/expressjs/express/blob/7e562c6d8daddff4604f8efaaf9db2cf98c6dcff/package.json#L56

I think that means this is still an issue?

Reply:
Yeah, this sounds right to me. Should be fixed in v5 though, so if this line is present in the 5.x docs we should remove it there.